src/components/volunteers/Volunteer/index.tsx

```diff
@@ -49,9 +85,23 @@ export default function Volunteer({
             {volunteer.previousOrganization?.name}
           </Typography>
         </Box>
-        <Typography variant="h5" mt={4}>
-          Role Verifications
-        </Typography>
+        <Box sx={{ display: 'flex', alignItems: 'center' }}>
+          <Typography variant="h5">Role Verifications</Typography>
+          <AddIcon sx={{ ml: 1 }} onClick={handleOpen} />
+          <Modal open={open} onClose={handleClose}>
+            <Box sx={{ bgcolor: 'background.paper' }}>
+              <RoleVerificationForm
+                roleVerificationData={formData}
+                onChange={(e) => {
+                  setFormData(e);
+                }}
+              />
+              <Button variant="contained" onClick={onSubmit}>
+                Submit
+              </Button>
+            </Box>
+          </Modal>
```

Comment:
this shouldnt be a modal, the form should appear above the role verification list

Reply:
disregard. instead of a modal tho use a `Dialog` component (they are essentially the same but a `Dialog` has some extra niceties around styling)